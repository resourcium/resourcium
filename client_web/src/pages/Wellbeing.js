import React from 'react';

export default function WellbeingPage() {
    return (
        <div className="WellbeingPage page">
            <h2>Wellbeing Page</h2>
        </div>
    );
}
