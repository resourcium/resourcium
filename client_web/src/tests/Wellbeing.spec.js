import { create } from 'react-test-renderer';
import React from 'react';

import Wellbeing from 'src/pages/Wellbeing';

import { wrapProvider } from 'src/tests/util';

test('greetings snapshot matches', () => {
    expect(create(wrapProvider(<Wellbeing />))).toMatchSnapshot();
});
