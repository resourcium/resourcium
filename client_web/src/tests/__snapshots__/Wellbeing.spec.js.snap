// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`greetings snapshot matches 1`] = `
<div
  className="WellbeingPage page"
>
  <h2>
    Wellbeing Page
  </h2>
</div>
`;
